grai-server/app/connections/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ConnectionConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
```

Comment:
This doesn't matter at all but we can probably get by with a simple AutoField for these.